com.unity.render-pipelines.universal/Runtime/DeferredLights.cs

```diff
@@ -291,6 +296,15 @@ public DeferredLights(Material tileDepthInfoMaterial, Material tileDeferredMater
             m_Tilers = new DeferredTiler[DeferredConfig.kTilerDepth];
             m_TileDataCapacities = new int[DeferredConfig.kTilerDepth];
 
+            GBufferDescriptors = new AttachmentDescriptor[GBufferSliceCount + 1]; // + Depth
+            GBufferDescriptors[0] = new AttachmentDescriptor(GetGBufferFormat(GBufferAlbedoIndex));
+            GBufferDescriptors[1] = new AttachmentDescriptor(GetGBufferFormat(GBufferSpecularMetallicIndex));
+            GBufferDescriptors[2] = new AttachmentDescriptor(GetGBufferFormat(GBufferNormalSmoothnessIndex));
```

Comment:
what about  `GBufferDescriptors[3]`?

Reply:
GBufferDescriptors[3] gets set in DeferredRenderer as it's the active camera color descriptor. will leave a comment to clarify that